.ci/get_workflow_status.py

```diff
@@ -0,0 +1,82 @@
+# coding: utf-8
+"""Get the most recent status of workflow for the current PR."""
+import json
+from os import environ
+from sys import argv, exit
+from time import sleep
+try:
+    from urllib import request
+except ImportError:
+    import urllib2 as request
```

Comment:
why was this necessary? I have a lot more experience with `requests` than `urllib`, so I haven't seen this pattern

Reply:
This for Python 2/3 compatibility. For the same reason I'm not using `with ...` statement but manually close connection. I know we dropped Python 2 support some time ago. But default installed Python version in `ubuntu-latest` container we run our actions in is **2**. I really don't want to install conda/new Python version/additional packages because I want to keep `Optional checks` workflow where this script is used as lightweight as possible with the aim of the fastest execution time. So this Python-agnostic and dependency-free solution will work fine on any default Python version that can be installed in container.

Reply:
ok I see, thanks. Fine with me